JSDemos/Demos/Scheduler/ContextMenuIntegration/description.md

```diff
@@ -1 +1,7 @@
-This demo shows how to create a context menu for appointments and cells using the **onAppointmentContextMenu** and **onCellContextMenu** functions.
\ No newline at end of file
+The [ContextMenu](/Documentation/Guide/UI_Components/ContextMenu/Overview/) allows you to replace a browser context menu with your custom context menu. The Scheduler offers the following properties to integrate this functionality:
+
+* [onAppointmentContextMenu](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#onAppointmentContextMenu)
+Sets a custom context menu for an appointment rectangle.
```

Comment:
Why do you prefer 'Set' over 'Specify'? As I see from our reviewers, they usually suggest 'Set a property to x' (same as assign x to y) and 'to specify' means to just add something in code. This is kind of example (not directly related) http://documentation.internalpages.devexpress.devx/documentation-guidelines/language-guidelines/misused-words/represent/#change-to-specifydefineidentify